transaction/std/src/memo_builder/gift_code_cancellation_memo_builder.rs

```diff
@@ -34,13 +39,18 @@ impl GiftCodeCancellationMemoBuilder {
         Self {
             gift_code_tx_out_global_index,
             wrote_change_memo: false,
+            fee: Amount::new(Mob::MINIMUM_FEE, Mob::ID),
         }
     }
 }
 
 impl MemoBuilder for GiftCodeCancellationMemoBuilder {
     /// Set the fee
-    fn set_fee(&mut self, _fee: Amount) -> Result<(), NewMemoError> {
+    fn set_fee(&mut self, fee: Amount) -> Result<(), NewMemoError> {
```

Comment:
This question has been raised before and the answer was "no" because these builders work in a way here they're called multiple places along the transaction building journey instead of configured all at once upfront.

Reply:
nope, our builders in this crate are not exactly following the rust pattern, and that's fine